forms/tests/test_api.py

```diff
@@ -248,10 +248,16 @@ def test_attachment_post(
         "ready": True,
     }
     response = admin_client.post(url, data=payload, content_type="application/json")
+    id = response.json()["id"]
 
     assert response.status_code == 201
     assert Attachment.objects.filter(answer=response.json()["id"]).exists()
 
+    url = reverse("answer-attachments", kwargs={"pk": id})
+    response = admin_client.get(url)
+    assert response.status_code == 200
+    assert len(response.json()) != 0
```

Comment:
Just a thought. Commonly when testing, you know exactly what the response will be. Therefore is a good idea to assert against things that you know that should be in the payload. fe. assert response.json()[0]['key'] == value